changelogs/fragments/1107-monit-fix-status-check.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - monit - fix modules ability to determine the current state of the monitored process (https://github.com/ansible-collections/community.general/pull/1107)
```

```suggestion
  - monit - fix modules ability to determine the current state of the monitored process (https://github.com/ansible-collections/community.general/pull/1107).
```
in accordance to the doc